scripts/maggie_scratch/leapfrog_frogger_testing.R

```diff
@@ -0,0 +1,144 @@
+rm(list = ls())
+library(tidyr)
+library(tidyverse)
+library(dplyr)
+setwd('C:/Users/mwalters/leapfrog/')
```

Comment:
This was mostly just for me so that I could see how I was testing it against spectrum without pulling in the spectrum file. It doesn't need to be pushed